hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -568,12 +579,15 @@ public long getPurgeIntervalNanos() {
    * @param methodName - RPC Request method name
    * @param details - Processing Detail.
    *
-   * if this request took too much time relative to other requests
-   * we consider that as a slow RPC. 3 is a magic number that comes
-   * from 3 sigma deviation. A very simple explanation can be found
-   * by searching for 68-95-99.7 rule. We flag an RPC as slow RPC
-   * if and only if it falls above 99.7% of requests. We start this logic
-   * only once we have enough sample size.
+   * if this request took too much time relative to other requests,
+   * and it took time exceed `logSlowRPCThresholdMs` we consider that as a slow RPC.
```

Comment:
nits: 

"if this request took too much time relative to other requests, and it took time exceed `logSlowRPCThresholdMs` we consider that as a slow RPC."
->

If a request took significant more time than other requests,
and its processing time is at least `logSlowRPCThresholdMs`, we consider this request as a slow RPC.

Reply:
Thanks @xinglin for your suggestion, i will update it later.